cmd/kubelet/kubelet.go

```diff
@@ -53,6 +54,7 @@ func run(command *cobra.Command) int {
 
 	command.SetGlobalNormalizationFunc(cliflag.WordSepNormalizeFunc)
 	if err := command.Execute(); err != nil {
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
```

Comment:
/lgtm

It's worth pointing out that this doesn't attempt to determine *how* the error should be printed (plain text or log entry), but that is good enough here. `cli.Run` on master handles that.